op-mode-definitions/container.xml.in

```diff
@@ -167,7 +167,7 @@
                 <path>container name</path>
               </completionHelp>
             </properties>
-            <command>if cli-shell-api existsActive container name "$4"; then sudo podman pull $(cli-shell-api returnActiveValue container name "$4" image); else echo "Container $4 does not exist"; fi</command>
+            <command>if cli-shell-api existsActive container name "$4"; then sudo podman pull --authfile /etc/containers/auth.json $(cli-shell-api returnActiveValue container name "$4" image); else echo "Container $4 does not exist"; fi</command>
```

Comment:
Please store the authfile in a volatile location like `/run/podman` so it's cleaned on reboot